flask_app/views.py

```diff
@@ -7,3 +9,20 @@
 @app.route("/")
 def index():
     return render_template("index.html", version=sys.version)
+
+
+@app.route("/secret")
+@login_required
+def secret():
+    return render_template("secret.html")
```

Comment:
Not sure this should be a part of the template. Maybe it belongs in a testing branch...